site/guides/resource-selectors/README.md

```diff
@@ -0,0 +1,85 @@
+## Resource Selectors
```

Comment:
Don't introduce a third type of content. Combination of the book and reference doc should cover everything.

The examples here should be part of 4.1 and 4.2. This is consistent with how we explain specifying image or functionConfig using render and eval.

Reply:
Initially we weren't sure about the length of the content for selectors and were afraid that it might disrupt the book flow.
But looking at the content, it feels, covering the use-case + description + example in the book might be okay.